db/index.js

```diff
@@ -0,0 +1,13 @@
+import pg from 'pg'
+
+const databaseUrl = process.env.DATABASE_URL
+
+if (undefined === databaseUrl) {
```

Comment:
Very very nitpicky comment, but this reads much better as `databaseUrl === undefined`. Code should always read how you are thinking about it logically, and so this would read better as `Is databaseUrl undefined?`.